pycbc/inference/models/analytic.py

```diff
@@ -189,3 +189,25 @@ def _loglikelihood(self):
         return 25 * (
             numpy.exp(-r/35) + 1 / (sigma * numpy.sqrt(2 * numpy.pi)) *
             numpy.exp(-0.5 * ((r - mu) / sigma) ** 2))
+
+class TestZeroLikelihood(BaseModel):
+    r"""The test distribution is zero likelihood.
+
+    Parameters
+    ----------
+    variable_params : (tuple of) string(s)
+        A tuple of parameter names that will be varied. Must have length 2.
+    **kwargs :
+        All other keyword arguments are passed to ``BaseModel``.
+
+    """
+    name = "test_zero_likelihood"
+
+    def __init__(self, variable_params, **kwargs):
+        # set up base likelihood parameters
+        super(TestZeroLikelihood, self).__init__(variable_params, **kwargs)
+
+    def _loglikelihood(self):
+        """Returns the log prior.
+        """
+        return self.logprior
```

Comment:
This should return 0, not logprior. If you return logprior for the `loglikelihood`, then you end up with `prior^2` as the log posterior (since `logprior` will also be called by `emcee_pt`, and via `logposterior` when using `emcee`).